local-cli/generator/templates/_flowconfig

```diff
@@ -35,13 +32,15 @@ node_modules/react-native/Libraries/react-native/react-native-interface.js
 [options]
 module.system=haste
 
+munge_underscores=true
```

Comment:
From: http://flowtype.org/docs/advanced-configuration.html

"set this to true to have Flow treat underscore-prefixed class properties and methods as private"

Sounds good!